deploy/helm/values.yaml

```diff
@@ -92,7 +92,7 @@ trivy:
   createConfig: true
 
   # imageRef the Trivy image reference.
-  imageRef: docker.io/aquasec/trivy:0.23.0
+  imageRef: docker.io/aquasec/trivy:0.24.2
```

Comment:
It's not the only place where we specify the default version of Trivy image. I'd suggest a dedicated PR to bump Trivy.

Reply:
Done, I will create a dedicated PR for the bump.